### ISSUE TITLE
Optimizer choice: Adam VS SGD

### ISSUE THREAD

**Issue by ConvMech** [zhengyang-wang/Deeplab-v2--ResNet-101--Tensorflow#48]
Dear Doctor Wang,

I noticed that DrSleep uses Adam for training, while yours and the original paper employed standard SGD. I am curious that do you have any experience on the performance difference of these two approach on this problem? 

Bests,

Xiong

**Comment by zhengyang-wang** [comment]
I believe all these projects you mentioned employ the ploy learning rate policy with weight decay.
